src/address.rs

```diff
@@ -1,23 +1,22 @@
 //! An address to an actor is a way to send it a message. An address allows an actor to be sent any
 //! kind of message that it can receive.
 
+use crate::envelope::{BroadcastEnvelopeConcrete, NonReturningEnvelope, ReturningEnvelope};
+use crate::refcount::{Either, RefCounter, Strong, Weak};
+use crate::send_future::ResolveToHandlerReturn;
+use crate::{inbox, Handler, KeepRunning, NameableSending, SendFuture};
+use event_listener::EventListener;
+use futures_core::{FusedFuture, Stream};
+use futures_sink::Sink;
+use futures_util::{future, FutureExt, StreamExt};
+use std::cmp::Ordering;
+use std::error::Error;
 use std::fmt::{self, Debug, Display, Formatter};
 use std::future::Future;
+use std::hash::{Hash, Hasher};
 use std::pin::Pin;
+use std::sync::Arc;
 use std::task::{Context, Poll};
```

Comment:
Nit: How imports are formatted is not entirely consistent across the crate. Some files group them into `std`, `extern`, `crate`. This was done here before and now they are in one block. I don't mind either way but consistency would be good :)

Reply:
Can we leverage rustfmt for this at all? I probably condensed it accidentally because when I used cargo fix to remove about 30 unused imports, it left behind blank lines

Reply:
You can but it is unfortunately an unstable formatting option: https://rust-lang.github.io/rustfmt/?version=v1.5.0&search=import#group_imports

There are at least three options:

1. Use nightly rustfmt to access the option. That is inconvenient because it requires you to install two toolchains
2. Use the command-line option: `cargo fmt -- --config group_imports=StdExternalCrate`. This works with stable Rust but would require an alias to be convenient.
3. Use something like [dprint](https://dprint.dev/) (spoiler: I am a fan) which has a rustfmt plugin that doesn't care about rust unstable options and lets you configure whatever you want.

To get started (2) is probably the easiest.